books/defsort/defsort.lisp

```diff
@@ -795,6 +795,13 @@ it has a special hack for that particular case.</p>
                     (consp x))
              :hints ((defsort-functional-inst
                        consp-of-comparable-mergesort
+                       ,subst1)))
+
+           (defthm ,(mksym prefix "-IS-IDENTITY-UNDER-SET-EQUIV")
+             (equal (consp (,sort x . ,extra-args))
+                    (consp x))
+             :hints ((defsort-functional-inst
+                       comparable-mergesort-under-set-equiv
```

Comment:
This doesn't seem like the right thorem body?

Reply:
@solswords You're correct. I've fixed this and I'm waiting for the regression to go through.

BTW, there are a number of lemmas directly preceding this which appear to be misnamed. As a concrete example, see examples.lisp which generates lemmas `string-no-duplicatesp-equal`, `string-true-listp`, `string-len` and `string-consp` which should all have `string-sort` in their names somewhere. I would submit a patch but it looks like there's code further down the line which depends on these awkward names.